test/basic_test.dart

```diff
@@ -399,6 +399,27 @@ void main() {
         expect(error.code, '420');
       }
     });
+
+    test('basic select table with converter', () async {
+      try {
+        await postgrestCustomHttpClient
+            .from('users')
+            .select()
+            .withConverter((data) => data)
+            .then<dynamic>(
+          (value) {
+            fail('Table was able to be selected, even tho it does not exist');
+          },
+          onError: (error) {
+            expect(error, isA<PostgrestException>());
+            expect(error.code, '420');
+          },
+        );
+      } on PostgrestException catch (error) {
+        expect(error.code, '420');
+      }
```

Comment:
Would we be able to do something like this instead?
```suggestion
        await postgrestCustomHttpClient
            .from('users')
            .select()
            .withConverter((data) => data);
            fail('Table was able to be selected, even tho it does not exist');
      } catch (error) {
        expect(error, isA<PostgrestException>());
        expect(error.code, '420');
      }
```